sdks/python/setup.py

```diff
@@ -90,6 +90,7 @@ def get_version():
     'httplib2>=0.8,<0.10',
     'mock>=1.0.1,<3.0.0',
     'oauth2client>=2.0.1,<4.0.0',
+    'protobuf==3.0.0',
```

Comment:
The problem is backwards incompatibility across minor versions. Datastore is using 3.0.0, so we are pinning to the same version.